thirdparty/README.md

```diff
@@ -670,14 +670,18 @@ Patches in the `patches` directory should be re-applied after updates.
 ## wslay
 
 - Upstream: https://github.com/tatsuhiro-t/wslay
-- Version: 1.1.1 (c9a84aa6df8512584c77c8cd15be9536b89c35aa, 2020)
+- Version: 1.1.1 (45d22583b488f79d5a4e598cc7675c191c5ab53f, 2021)
```

Comment:
Should we change the version number to: `1.1.1+git` then?
The actual `1.1.1` release is the commit we are leaving behind: https://github.com/tatsuhiro-t/wslay/tree/release-1.1.1

Reply:
Technically yeah, I just felt that we're still so close to `1.1.1` that it's fine to keep it as human readable version. But yeah `1.1.1+git` is a good idea, I'll use that.